src/openapi/components/schemas/BulkUploadTask.json

```diff
@@ -29,6 +29,28 @@
 			"$ref": "./File.json",
 			"readOnly": true
 		},
+		"attachmentsArchiveFileId": {
+			"oneOf": [
+				{
+					"type": "integer",
```

Comment:
It looks like we have an `id.json` type but haven't been using it (guilty myself here too)

Reply:
Oh great catch yes!